src/Microsoft.DotNet.Build.Tasks/PackageFiles/packages.Unix.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="dnx-mono" version="1.0.0-rc2-16128" />
```

Comment:
If you are using dotnet cli why do you need to restore dnx? Just curious :smile: